packages/@okta/vuepress-site/guides/sign-into-spa/section03/react/attachcomponentsetc.md

```diff
@@ -1,18 +1,112 @@
 ## Attach Components to the Secure Router
 
-You need to provide these routes in your sample application so that we can sign the user in and handle the callback from Okta. We show you how to set these up below using [React Router DOM](https://github.com/ReactTraining/react-router/tree/master/packages/react-router-dom):
+We provide the `Security` component which makes an Auth object available to child components and the `withAuth` HOC. We show you how to set these up below using [React Router DOM](https://github.com/ReactTraining/react-router/tree/master/packages/react-router-dom):
 
-- `/`: A default home page to handle basic control of the app.
-- `/implicit/callback`: This is where auth is handled for you after redirection.
+```javascript
+import React, { Component } from 'react';
+import { BrowserRouter as Router, Route } from 'react-router-dom';
+// import config from ...
+
+class App extends Component {
+  render() {
+    return (
+      <Router>
+        <Security {...config} >
+          { /* child components have access to auth context */ }
```

Comment:
I'd provide an example of having "access to auth context" here, even if the example is commented out and not runnable by itself.